climada/engine/impact_forecast.py

```diff
@@ -88,3 +88,66 @@ def from_impact(
                 imp_mat=impact.imp_mat,
                 haz_type=impact.haz_type,
             )
+
+    @property
+    def at_event(self):
+        return self._at_event
+
+    @at_event.setter
+    def at_event(self, value):
+        """Set the total exposure value close to a hazard"""
```

Comment:
What does that mean? This is not what `at_event` does. 
```suggestion
        """Set the total exposure value close to a hazard"""
```